proto/storage/config.proto

```diff
@@ -62,7 +62,19 @@ message ReportRetentionConfig {
   uint32 downloadable_report_global_retention_bytes = 3;
 }
 
-// next available tag:9
+// next available tag: 2
+message PrometheusMetrics {
+  message MetricGroup {
+    message Labels {
+      repeated string labels = 1;
+    }
+    uint32 gathering_period_minutes = 1;
+    map<string, Labels> metrics = 2;
```

Comment:
Not sure about the `metrics` field name, as there is another `metrics` field inside.